src/main/datalog_console/client.cljs

```diff
@@ -30,15 +30,24 @@
   (let [port devtool-port]
     (.addListener (gobj/get port "onMessage")
                   (fn [msg]
-                    (when-let [db-str (gobj/getValueByKeys msg ":datalog-console.remote/remote-message")]
-                      (reset! rconn (d/conn-from-db (cljs.reader/read-string db-str))))))
+                    (when-let [remote-message (cljs.reader/read-string (gobj/getValueByKeys msg ":datalog-console.remote/remote-message"))]
+                      (cond
+                        (d/db? remote-message)
+                        (reset! rconn (d/conn-from-db remote-message))
+
+                        (= :success remote-message)
```

Comment:
This still has to change. Need to think of a better pattern for handling different types of results when interacting with remote databases. Perhaps adopting HTTP status responses is the best approach?